src/Configuration/AccountManager.cs

```diff
@@ -9,42 +10,64 @@ internal static class AccountManager
     {
         public static List<Account> Accounts;
 
-        private static void Load(string serverName)
+        public static string[] GetAccountNames(string serverName)
         {
-            if (Accounts == null)
+            try
             {
-                var accounts = LoadAccountsFromFile();
-                Accounts = accounts.Where(x => x.Server == serverName).ToList();
+                Load(serverName);
+                return Accounts?.Where(x => x.Server == serverName).Select(x => x.UserName).ToArray() ?? new string[] { };
```

Comment:
I see this a lot as well, `x` instead of a proper name for what x might be. Why not `account`? And in case the line gets "too long", you can break the line before `.Select` ;)

Reply:
bad habits die hard :)

Reply:
changed.